src/components/project/form-row.vue

```diff
@@ -0,0 +1,100 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <tr class="form-row">
+    <!--todo: come back and figure out all the permissions-->
+    <td class="name">
+      <link-if-can :to="primaryFormPath(form)">
+        {{ form.nameOrId() }}
+      </link-if-can>
+    </td>
+    <td class="state-received">
+      {{ form.reviewState.received }} recieved
+    </td>
+    <td class="state-edited">
+      {{ form.reviewState.edited }} edited
+    </td>
+    <td class="state-has-issues">
+      {{ form.reviewState.hasIssues }} has issues
+    </td>
+    <td class="last-submission">
+      <div v-if="form.lastSubmission != null">
+        <router-link :to="submissionsPath">
+          <i18n :tag="false" path="common.lastSubmission">
+            <template #dateTime>
+              <date-time :iso="form.lastSubmission"/>
+            </template>
+          </i18n>
+        </router-link>
+      </div>
+    </td>
+    <td class="total-submissions">
+      <div v-if="form.publishedAt != null">
+        <router-link :to="submissionsPath">
+          <span>{{ $tcn('count.submission', form.submissions) }}</span>
+          <span class="icon-angle-right"></span>
+        </router-link>
+      </div>
+    </td>
+  </tr>
+</template>
+
+<script>
+import DateTime from '../date-time.vue';
+import LinkIfCan from '../link-if-can.vue';
+import Form from '../../presenters/form';
+import routes from '../../mixins/routes';
+import { requestData } from '../../store/modules/request';
+
+export default {
+  name: 'ProjectFormRow',
+  components: { DateTime, LinkIfCan },
+  mixins: [routes()],
+  props: {
+    form: {
+      type: Form,
```

Comment:
In anticipation of a change I'll be making soon, let's go ahead and make this `Object` rather than `Form`! Soon it's not going to be possible to import presenter classes or use them as types.